anybox/recipe/odoo/server.py

```diff
@@ -183,15 +183,19 @@ def _create_gunicorn_conf(self, qualified_name):
 
     def _get_server_command(self):
         """Return a full path to the main Odoo server command."""
-        return join(self.openerp_dir, 'openerp-server')
+        if major_version(self.version_detected)[0] >= 10:
+            base_name = 'odoo-bin'
+        else:
+            base_name = 'openerp-server'
+        return join(self.odoo_dir, base_name)
```

Comment:
I'm afraid compatibility with older versions was completely dropped initially with this change. @archetipo do you plan on fixing this?